ios/ios/SplashView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SplashView.swift
+//  ios
+//
+//  Created by Thiên Nguyễn on 8/10/23.
+//  Copyright © 2023 orgName. All rights reserved.
+//
+
+import SwiftUI
+
+let SHOW_LOGO_DELAY = 0.5
+let SHOW_LOGO_DURATION = 1.0
```

Comment:
In Swift convention, we use lowerCamelCase for naming variables; these should be too. 

Besides, Ideally your global constants will be located within an `enum`, `extension`, or `struct` (just like `Fonts`).

If you still want to keep this approach, please consider setting it with the `private`  keyword.

```suggestion
private let showLogoDelay = 0.5
private let showLogoDuration = 1.0
```